cors/mod.ts

```diff
@@ -0,0 +1,207 @@
+import { Drash } from "../deps.ts";
+import { vary } from "./deps.ts";
+
+interface ArrayOfValueOrArray<T> extends Array<ValueOrArray<T>> {}
+type ValueOrArray<T> = T | ArrayOfValueOrArray<T>;
+type OriginOption =
+  | ((
+    request: Drash.Http.Request,
+    response?: Drash.Http.Response,
+  ) => string | Promise<string>)
+  | ValueOrArray<string | boolean | RegExp>;
```

Comment:
Should the request and response be a part of this type?

Reply:
https://github.com/drashland/deno-drash-middleware/pull/1/commits/1627c064bf674eeeccbe726cd79bd685da78dd8b